tests/cucumber/features/menu/edit/reset-clone-undo-redo.feature

```diff
@@ -45,4 +45,4 @@ Feature: User can create a material, then undo the changes and/or reset to the i
      And material with following data does not exist in state
       | path    | index   |
       | si-clone.json | $INT{2} |
-
+    When I undo
```

Comment:
Remove

Reply:
Oops. That's probably slipped during cmd+F search